apps/test/unit/code-studio/components/teacher-homepage/CourseCardTest.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+import  CourseCard from '@cdo/apps/templates/teacherHomepage/CourseCard';
```

Comment:
The test and src files should have the same paths as much as possible, i.e. the test path should probably be
`apps/test/unit/templates/teacherHomepage/CourseCardTest.js`